src/app/component/MissionDetailsCard.jsx

```diff
@@ -0,0 +1,384 @@
+import React from "react";
```

Comment:
Can we break this file into smaller ones? Example: MissionDetailsStatus , MissionDetailsIconList  etc. could live in their own file.